File: app/src/main/java/br/fmu/bioreino/util/ListaUtil.java
package br.fmu.bioreino.util;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;

import br.fmu.bioreino.R;

public class ListaUtil {

    public static View inflaLayout(@NonNull ViewGroup parent, int layout) {
        return LayoutInflater.from(parent.getContext()).inflate(layout, parent, false);
    }

    public static View mensagemVazia(String mensagem, Context context, ViewGroup viewGroup) {
        View view = LayoutInflater.from(context).inflate(R.layout.layout_lista_vazia, viewGroup, false);
        TextView mensagemView = view.findViewById(R.id.layout_lista_vazia_mensagem);
        mensagemView.setText(mensagem);
        return view;
    }

    public static int formataProgressoBarra(int progresso) {
        if (progresso == 0 || progresso > 10) {
            return progresso;
        }
        return 10;
    }

}

File: app/src/main/java/br/fmu/bioreino/controller/util/ListaUtil.kt
package br.fmu.bioreino.controller.util

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import br.fmu.bioreino.R

object ListaUtil {
    @JvmStatic
    fun inflaLayout(parent: ViewGroup, layout: Int): View {
        return LayoutInflater.from(parent.context).inflate(layout, parent, false)
    }

    fun mensagemVazia(mensagem: String?, context: Context?, viewGroup: ViewGroup?): View {
        val view =
            LayoutInflater.from(context).inflate(R.layout.layout_lista_vazia, viewGroup, false)
        val mensagemView = view.findViewById<TextView>(R.id.layout_lista_vazia_mensagem)
        mensagemView.text = mensagem
        return view
    }

    @JvmStatic
    fun formataProgressoBarra(progresso: Int): Int {
        return if (progresso == 0 || progresso > 10) {
            progresso
        } else 10
    }
}